include/irr/asset/IAsset.h

```diff
@@ -140,13 +188,18 @@ class SAssetBundle
     //! Only valid if isInAResourceCache() returns true
     std::string getCacheKey() const { return m_cacheKey; }
 
+	//! Getting size of an Asset stored by m_contents
     size_t getSize() const { return m_contents->size(); }
+	//! Checking if an Asset stored by m_contents is empty
     bool isEmpty() const { return getSize()==0ull; }
 
+	//! Overloaded operator checking if both Assets\b are\b the same objects in memory
```

Comment:
Bundles are indentical if array pointers match, not if contents match (makes dealing with them a lot faster)